vendor/licenses/git_submodule/vscode-dascript.dep.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: vscode-dascript
+version: 9ff4e434480cc977eeb5e9ee7bb4d144bcc2e4d2
+type: git_submodule
+homepage: https://github.com/GaijinEntertainment/vscode-dascript.git
+license: none
+licenses: []
+notices: []
```

Comment:
We can not accept unlicensed grammars. The grammar repo needs one of [these licenses](https://github.com/github-linguist/linguist/blob/9b1023ed5d308cb3363a882531dea1e272b59977/vendor/licenses/config.yml#L4-L15) in order for us to accept it.